lib/Echidna/Config.hs

```diff
@@ -73,21 +73,25 @@ instance FromJSON EConfig where
                           <*> v .:? "shrinkLimit" .!= 5000
                           <*> pure Nothing
                           <*> v .:? "seed"
-        names = const $ const mempty :: Names
-        ppc = cc <&> \c x g -> runReader (ppCampaign x) (c, names) ++ "\nSeed: " ++ show g
+        names :: Names
+        names Sender = (" from: " ++) . show
+        names _      = const ""
+        ppc :: Y.Parser (Campaign -> Int -> String)
+        ppc = liftM2 (\cf xf c g -> runReader (ppCampaign c) (cf, xf, names) ++ "\nSeed: " ++ show g) cc xc
         style :: Y.Parser (Campaign -> Int -> String)
         style = v .:? "format" .!= ("text" :: String) >>=
           \case "text"             -> ppc
                 "json"             -> pure . flip $ \g ->
                   unpack . encode . set (_Object . at "seed") (Just . toJSON $ g) . toJSON;
-                "none"             -> pure . const . const $ ""
+                "none"             -> pure $ \_ _ -> ""
                 _                  -> pure $ \_ _ -> M.fail
                   "unrecognized ui type (should be text, json, or none)" in
     EConfig <$> cc
             <*> pure names
             <*> (SolConf <$> v .:? "contractAddr"   .!= 0x00a329c0648769a73afac7f9381e08fb43dbea72
                          <*> v .:? "deployer"       .!= 0x00a329c0648769a73afac7f9381e08fb43dbea70
-                         <*> v .:? "sender"         .!= [0x00a329c0648769a73afac7f9381e08fb43dbea70]
+                         <*> v .:? "sender"         .!= [ 0x00a329c0648769a73afac7f9381e08fb43dbea70
```

Comment:
I suggest to make these addresses:

(1) smaller, so they are easy to ready and
(2) very different between each other, so they are easier to read.